frame/bags-list/remote-tests/Cargo.toml

```diff
@@ -15,7 +15,7 @@ targets = ["x86_64-unknown-linux-gnu"]
 [dependencies]
 # frame
 pallet-staking = { path = "../../staking", version = "4.0.0-dev" }
-pallet-bags-list = { path = "../../bags-list", version = "4.0.0-dev" }
+pallet-bags-list = { path = "../../bags-list", version = "4.0.0-dev", features = ["fuzz"] }
```

Comment:
@kianenigma this is the change I am not a fan of, perhaps we can rename this feature to something like remote-tests or external-tests. Or keep as is.

Reply:
I think it is fine.